tests/utils.go

```diff
@@ -1686,8 +1686,15 @@ func removeNamespaces() {
 	fmt.Println("")
 	for _, namespace := range testNamespaces {
 		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespace)
-		EventuallyWithOffset(1, func() bool { return errors.IsNotFound(virtCli.CoreV1().Namespaces().Delete(namespace, nil)) }, 240*time.Second, 1*time.Second).
-			Should(BeTrue())
+
+		EventuallyWithOffset(1, func() error {
+			err = virtCli.CoreV1().Namespaces().Delete(namespace, nil)
+			if err != nil {
+				return err
+			}
+			return fmt.Errorf("namespace %s still exists", namespace)
```

Comment:
This is obviously wrong because a deletion with a nil error is a success delivery of the request to the API.
If we take a few steps back, I do not understand why a delete instruction is retried.

If we look how the VMI deletion was handled, the retry is on the "Get" while the "Delete" is checked only once.

Reply:
changed. I Think now it is better